.github/workflows/release-checklist.yml

```diff
@@ -0,0 +1,25 @@
+name: Plugin Release Build
+
+on:
+  pull_request:
+    types:
+      - opened
+    branches:
+      - 'trunk'
+
+jobs:
+  checklist_job:
+    if: ${{ startsWith( github.head_ref, 'release/' ) }}
+    runs-on: ubuntu-latest
+    name: Sensei release checklist
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Checklist
+        uses: wyozi/contextual-qa-checklist-action@master
```

Comment:
Thank you for implementing it! 🎉

I'm just adding here the same comment I added to the Sensei Pro:

Since this action is not widely used, maybe we could fork it before using it. Just to make sure we won't have malicious code doing anything here. WDYT?

We did the same for another action we added. See it here: https://github.com/Automattic/sensei/pull/5233#issuecomment-1156753101

Reply:
Done in [77637dd](https://github.com/Automattic/sensei/pull/5309/commits/77637dd3e59f7c1c8891baa4539b4cff7b125ebd)

Reply:
Thank you @gkaragia !